main.go

```diff
@@ -48,10 +48,13 @@ var DefaultFuncs = template.FuncMap{
 	"stringSlice": func(s ...string) []string {
 		return s
 	},
+	"inc": func(i int) int {
```

Comment:
unused + I'm pretty sure it already exists in the go template system

Reply:
done